python/cuml/datasets/blobs.py

```diff
@@ -133,6 +133,12 @@ def make_blobs(n_samples=100, n_features=2, centers=None, cluster_std=1.0,
     --------
     make_classification: a more intricate variant
     """
+
+    # Set the default output type to "cupy". This will be ignored if the user
+    # has set `cuml.global_output_type`. Only necessary for array generation
+    # methods that do not take an array as input
+    cuml.internals.set_api_output_type("cupy")
```

Comment:
Yes this can be a little weird but it works on a stack where the lowest caller wins. So the hierarchy is: user set value > outer function > inner function. So in this case, if the user has set an output type, this will have no effect. But if the user has left `output_type=None`, then this value will be used.